src/stories/HeatmapVis.stories.tsx

```diff
@@ -8,14 +8,12 @@ import HeatmapVis, {
 } from '../h5web/visualizations/heatmap/HeatmapVis';
 import { ScaleType } from '../h5web/visualizations/shared/models';
 import { INTERPOLATORS } from '../h5web/visualizations/heatmap/interpolators';
-import { getDomain, mockValues } from '../packages/lib';
-import { getMockDatasetDims } from '../h5web/providers/mock/utils';
-
-// Prepare 2D data array
-const values = mockValues.twoD.flat(Infinity) as number[];
-const dataArray = ndarray(values, getMockDatasetDims('twoD'));
-const domain = getDomain(values);
-const logSafeDomain = getDomain(values, ScaleType.Log);
+import { getDomain } from '../packages/lib';
```

Comment:
Should it import from `lib` here ?

Reply:
Meh, doesn't matter here. We'll switch it to `@h5web/lib` later anyway for #358.